src/lib/bootstrap_controller/bootstrap_controller.ml

```diff
@@ -302,45 +360,14 @@ end = struct
       ; current_root= initial_root_transition }
     in
     let transition_graph = Transition_cache.create () in
+    let root_sync_ledger =
+      Root_sync_ledger.create ledger_db ~logger:t.logger ~trust_system
+    in
+    ( if not should_ask_best_tip then Deferred.unit
```

Comment:
Not that it matters much, but you could have also done `(if ... then ... |> don't_wait_for) ;` instead.